tests/data/cred_repo/setup

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+tag="pytest [credential verification]:"
+
+# verifying proper operation for copying credential for image creation.
+# so '/workspace/tools/user_config/auth.d/' must to there to place .conf file
+
+dir=$1
+uid=$2
+gid=$3
+
+function setup_dir()
+{
+    mkdir -p $1
+    chown -R ${uid}:${gid} $1
+}
+
+echo "$tag init operation called"
+
+[[ ! -d "/workspace/tools/user_config/auth.d/" ]] && setup_dir "/workspace/tools/user_config/auth.d/" 
+[[ ! -d "/workspace/state" ]] && setup_dir "/workspace/state"
+
+which debootstrap > /dev/null
+if [ $? -ne 0 ]; then
+    echo "$tag installing debootstrap..."
+    apt install debootstrap -y
+fi
+
+cp -fr ${dir}/keys/* /etc/apt/trusted.gpg.d/
+sync
```

Comment:
This does not make any sense

 1. Don't use a shell script
 2. Don't install debootstrap, it exists already in the container. If not the whole root_generator will not work
 3. Don't copy any keys to the containers trusted keys, that is just not required because it will never be used by the tools
 4. Don't use sync here, that doesn't make any sense
 5. Don't create the state dir, just assume it exists. If not a working fallback is used